dashboard/app/views/courses/index.html.haml

```diff
@@ -0,0 +1 @@
+%h1 Course Page
```

Comment:
Is this the right action/file name for an individual course page, or should index be reserved for a page that asks for all courses, and course/some_course goes somewhere else?